packages/micromark-util-events-to-acorn/dev/index.js

```diff
@@ -146,7 +146,28 @@ export function eventsToAcorn(events, options) {
 
   if (estree) {
     // @ts-expect-error: acorn *does* allow comments
-    estree.comments = comments
+    estree.comments = comments.map((comment) => {
+      assert('start' in comment, 'expected `start` in comment from acorn')
+      assert('end' in comment, 'expected `end` in comment from acorn')
+
+      const pointStart = parseOffsetToUnistPoint(comment.start)
+      const pointEnd = parseOffsetToUnistPoint(comment.end)
+      comment.start = pointStart.offset
+      comment.end = pointEnd.offset
+      comment.loc = {
+        start: {
+          line: pointStart.line,
+          column: pointStart.column - 1,
```

Comment:
<img width="371" alt="image" src="https://user-images.githubusercontent.com/8336744/162178942-ee95be05-790e-4755-882e-3b1787368432.png">

Seems incorrect, let me check 😅

Reply:
It seems this lib is not called at all for these comments. @wooorm Where do you think could be related else? 🤣